website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc

```diff
@@ -0,0 +1,176 @@
+It's exciting to see the growth of this project through the improvements made to FreeBSD, as well as the increase in users and contributors. 
+
+Over the past few quarters, we've built up our technology, advocacy, and partnership teams to accelerate our work in improving the operating system, increasing the adoption and visibility of FreeBSD, and increasing the number of partners who help fund our work.
+
+Below you'll find updates from each team to see the work we've accomplished to support you, the community and the operating system we all love.
+But, first I want to share a fundraising update.
+Last quarter we raised $41,154 towards our goal of raising over $2,000,000.
+You can see our 2024 budget to understand how we are spending your donations here: link:https://freebsdfoundation.org/wp-content/uploads/2024/03/Budget2024-Approved-Public.pdf[].
+
+Over half the budget goes directly into improving and securing FreeBSD.
+If there is a security vulnerability out there, we have software developers on staff who can quickly step in, evaluate the situation, and put in a change or workaround if needed.
+We have a full-time developer who leads the continuous integration efforts and investigates ways to improve the tools to test code, improve test coverage, and help developers be more efficient.
+We've also allocated more funding towards our advocacy efforts.
+This includes creating content to highlight FreeBSD's strengths and differentiators, talking to commercial users and documenting their use cases, and promoting the work you are doing.
+
+Please consider funding our efforts to help keep FreeBSD innovative, secure, and stable by making a donation here: link:https://freebsdfoundation.org/donate/[].
+
+If you are a corporate user, please consider becoming a partner!
+Go here to find out more information about our partnership opportunities: link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[].
+
+==== OS Improvements
+During the second quarter of 2024, 204 src, 50 ports, and 11 doc tree commits identified The FreeBSD Foundation as a sponsor.
+
+The Foundation is sponsoring 13 projects.
+
+* mailto:christos@FreeBSD.org[Christos Margiolis] continued to improve FreeBSD's audio stack and provide audio developers with useful tools and frameworks to facilitate sound development on FreeBSD.
+Refer to the <<_audio_stack_improvements,Audio Stack Improvements>> entry for details.
+
+* mailto:pjd@FreeBSD.org[Pawel Dawidek] is in the final stages of a project to add hierarchical rate limits to OpenZFS.
+The configured limits for bandwidth and operations will work like quotas.
+If a child dataset is configured with a higher limit than its parent, the parent's lower limit will be enforced.
+For details, refer to Pawel's link:https://github.com/openzfs/zfs/pull/16205[pull request in the OpenZFS repository].
```

Comment:
Noting here that pjd@ submitted a report that still has to be committed. When it is, do you mind if we link to it from here, perhaps using "a project to add hierarchical rate limits to OpenZFS" as the link text?

Reply:
Sounds good.  Thanks.

Reply:
@pauamma, could you share the title of Pawe's report entry?  That way, I can try to save you a bit of work.

Reply:
"Hierarchical rate limits for OpenZFS"